_gpon/vendor.md

```diff
@@ -38,8 +44,10 @@ Here is a list of the most popular Vendor IDs:
 | `TMBB` | `544d4242` | Technicolor          |
 | `TPLG` | `54504c47` | TP-Link              |
 | `UBNT` | `55424e54` | Ubiquiti             |
+| `UGRD` | `55475244` | UGrid                |
 | `YHTC` | `59485443` | Youhua               |
 | `ZTEG` | `5a544547` | ZTE                  |
 | `ZYWN` | `5a59574e` | Zyxel                |
+| `ZYXE` | `5a595845` | Zyxel                |
```

Comment:
it should be specified which of the two zyxels is more common and/or where it is used (Example: EMEA ZYWN and America ZYXE, etc.)

Reply:
I don't know which is more common, but they use ZYXE in some markets.
I have ZyXEL PMG4506 / 512NP, they are using ZYXE.